### ISSUE TITLE
EXIF 2.3 values unreliable

### ISSUE THREAD

**Issue by qligier** [electerious/Lychee#481]
Hi,

The PHP function `exif_read_data` [seems to have issues](http://stackoverflow.com/questions/30590310/php-exif-read-data-returns-wrong-and-incomplete-data) with EXIF 2.3, the return values are unreliable.

Compare the result of `exif_read_data($url, 'EXIF', 0)`:
```text
Array
(
    [Copyright] => tin Ligier
    [DateTime] => 1 (Windows)
    [Make] => 
    [Model] => CORPORATION
    ...
)
```

With `$image = new Imagick($url); $image->getImageProperties("exif:*")`:
```text
Array
(
    [exif:Copyright] => Quentin Ligier
    [exif:DateTime] => 2016:03:14 12:37:20
    [exif:Make] => NIKON CORPORATION
    [exif:Model] => NIKON D7100
    ...
)
```

This last method is accurate but also very slow, as it loads the whole image into memory (a quick time measurement gives 0.2 second...). It could be possible to store the EXIF and IPTC values into database and this would allow to set custom values (as requested in #301).

Another possibility would be to use command line to get the EXIF values, but it would require accessing the `shell_exec` function and parsing the result.

My system information:
```
Lychee Version:  3.0.9
DB Version:      030003
System:          Linux
PHP Version:     5.6
MySQL Version:   50547
Imagick:         1
Imagick Active:  1
Imagick Version: 1673
GD Version:      2.1.1-dev
```

Regards,
Quentin

**Comment by electerious** [comment]
Is this a known PHP problem? Is this issue still present in PHP 7? `exif_read_data` works fine so far, would really like to keep it.

**Comment by qligier** [comment]
Hi,

The [issue is known](https://bugs.php.net/bug.php?id=50845) but nothing has been done in 6 years.
The issue seems to occur only with [photos that are exported through Lightroom 6.0.1](https://forums.adobe.com/thread/1835290), it has been solved in Lightroom 6.2.

I'll update the software and reupload the photos.

Quentin

**Comment by qligier** [comment]
Updating the software and re-uploading the photos solved the problem.

**Comment by qligier** [comment]
[The issue](https://bugs.php.net/bug.php?id=50845) has been resolved in PHP 7.0.9 so updating should also be a solution.
